.circleci/build-assets.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+echo "Begin build for $CIRCLE_ENV${PR_ENV:+ for }$PR_ENV. Pantheon test environment is $TERMINUS_SITE.$TERMINUS_ENV"
+
+# Build assets with Composer
+composer -n build-assets
+
+# Compile Sass or run any other build steps necessary
```

Comment:
I also think we should avoid files like this. An overarching metric of quality for these repos that Josh and I discussed is the number of files/changes that separate our repos from the community-consensus example repos of https://github.com/roots/bedrock and https://github.com/drupal-composer/drupal-project

Every file we add to these repos further complicates the goal of Pantheon + Composer being a light addition to those community consensus repos.

For now, I think we should stick with consistency between our examples stick with `build-assets` as the command to modify. (Though I'd eventually like to see that removed too in favor of just adding on to `post-install-cmd` directly)